src/components/TradeWidget/TokenRow.tsx

```diff
@@ -176,17 +177,20 @@ const TokenRow: React.FC<Props> = ({
   const balanceClassName = !error && sellAmountOverMax ? 'warning' : 'success'
   const inputClassName = error ? 'error' : sellAmountOverMax ? 'warning' : ''
 
+  const fullOverMaxAmount = overMax.gt(ZERO) && formatAmountFull({ amount: overMax, precision: selectedToken.decimals })
```

Comment:
Consider
```suggestion
  const fullOverMaxAmount = overMax.gt(ZERO) ? formatAmountFull({ amount: overMax, precision: selectedToken.decimals }) : ''
```
so that you don't have to `fullOverMaxAmount as string`